presto-benchmark-runner/src/test/java/com/facebook/presto/benchmark/BenchmarkTestUtil.java

```diff
@@ -0,0 +1,154 @@
+import java.util.Map;
+import java.util.Optional;
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+
+public class BenchmarkTestUtil
+{
+    public static final String CATALOG = "benchmark";
+    public static final String SCHEMA = "default";
+    public static final String XDB = "presto";
+    public static final String BENCHMARK_SUITES_TABLE = "benchmark_suites";
+    public static final String BENCHMARK_QUERIES_TABLE = "benchmark_queries";
+
+    private static final MySqlOptions MY_SQL_OPTIONS = MySqlOptions.builder()
+            .setCommandTimeout(new Duration(90, SECONDS))
+            .build();
+
+    private BenchmarkTestUtil()
+    {
+    }
+
+    public static TestingMySqlServer setupMySql()
+            throws Exception
+    {
+        TestingMySqlServer mySqlServer = new TestingMySqlServer("testuser", "testpass", ImmutableList.of(XDB), MY_SQL_OPTIONS);
+        try (Handle handle = getJdbi(mySqlServer).open()) {
+            handle.execute("CREATE TABLE benchmark_suites (\n" +
+                    "  id bigint(20) unsigned NOT NULL AUTO_INCREMENT,\n" +
+                    "  suite varchar(256) NOT NULL,\n" +
+                    "  query_set varchar(256) NOT NULL,\n" +
+                    "  phases mediumtext NOT NULL,\n" +
+                    "  session_properties mediumtext NOT NULL,\n" +
+                    "  created_by varchar(256) NOT NULL,\n" +
+                    "  create_time timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,\n" +
+                    "  update_time timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,\n" +
+                    "  PRIMARY KEY (id),\n" +
+                    "  UNIQUE KEY suite (suite))");
+
+            handle.execute("CREATE TABLE benchmark_queries (\n" +
+                    "  id bigint(20) unsigned NOT NULL AUTO_INCREMENT,\n" +
+                    "  query_set varchar(256) NOT NULL,\n" +
+                    "  name varchar(256) NOT NULL,\n" +
+                    "  catalog varchar(256) NOT NULL,\n" +
+                    "  benchmark_schema varchar(256) NOT NULL,\n" +
+                    "  query mediumtext NOT NULL,\n" +
+                    "  create_time timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,\n" +
+                    "  update_time timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,\n" +
+                    "  PRIMARY KEY (id),\n" +
+                    "  UNIQUE KEY query_set_name (query_set,name))");
+        }
+        return mySqlServer;
+    }
+
+    public static Jdbi getJdbi(TestingMySqlServer mySqlServer)
+    {
+        return Jdbi.create(mySqlServer.getJdbcUrl(XDB)).installPlugin(new SqlObjectPlugin());
+    }
+
+    public static void insertBenchmarkQuery(Handle handle, String querySet, String name, String query)
+    {
+        handle.execute(
+                "INSERT INTO benchmark_queries(\n" +
+                        "    query_set, name, catalog, benchmark_schema, query, create_time, update_time)\n" +
```

Comment:
You don't need to insert into `create_time` and `update_time`. They'll be populated with default values.